src/current/_includes/releases/v23.1/v23.1.11.md

```diff
@@ -0,0 +1,141 @@
+- Fixed an issue with [the "full scan" filter in the UI]({% link v23.1/ui-sql-dashboard.md %}#full-table-index-scans), where the filter was not returning any results. [cockroachdb/cockroach#109274][#109274]
+- Going to the [Transaction Details Page](https://www.cockroachlabs.com/docs/v23.1/ui-transactions-page#transaction-details-page) from [Workload Insights > High Contention](https://www.cockroachlabs.com/docs/v23.1/ui-insights-page#high-contention) is fixed. Previously, the link would not show any results. [#109253][#109253]
+- Fixed the column names on the [`SELECT` queries]({% link v23.1/selection-queries.md %}) against the tables `crdb_internal.node_txn_execution_insights` and `crdb_internal.cluster_txn_execution_insights` during the [creation of debug.zip files]({% link v23.1/cockroach-debug-zip.md %}). [cockroachdb/cockroach#109515][#109515]
+- Fixed a bug that could cause some rows to be silently skipped during [`IMPORT`](../v23.1/import.html) when a node failed. [#109663][#109663]
+- Fixed a bug in [geospatial queries]({% link v23.1/spatial-data-overview.md %}) where more rows could be returned by the query than expected. This could happen when a query filter of the form `ST_Distance(geog1, geog2) > constant` or `ST_MaxDistance(geom1, geom2) > constant`, where the operator was one of `>`, `<`, `>=`, `<=`, or a filter of the form `ST_Distance(geog1, geog2, false) = 0` would sometimes mistakenly evaluate to `true` when one or both of the inputs was `NULL` or an empty [geography]({% link v23.1/architecture/glossary.md %}#geography) or [geometry]({% link v23.1/architecture/glossary.md %}#geometry). [#109393][#109393]
+- Fixed an issue where a [split](../v23.1/architecture/distribution-layer.html#range-splits) can be called on an invalid key that's in the form of `someValidKey.Next()` during [restore](../v23.1/restore.html). This split key can land in the middle of a row with [column families](../v23.1/column-families.html), and thus result in failing SQL queries when querying the restored table. [#109777][#109777]
+- Fixed an issue where a [split](../v23.1/architecture/distribution-layer.html#range-splits) can be called on an invalid key that's in the form of `someValidKey.Next()` during [restore](../v23.1/restore.html) with the `bulkio.restore.use_simple_import_spans` [cluster setting]({% link v23.1/cluster-settings.md %}) set to `true`. This split key can land in the middle of a row with [column families](../v23.1/column-families.html), and thus result in failing SQL queries when querying the restored table. [#109940][#109940]
+- The `difference` [built-in function]({% link v23.1/functions-and-operators.md %}) had its return type incorrectly set to [`string`]({% link v23.1/string.md %}) instead of [`integer`]({% link v23.1/int.md %}). [cockroachdb/cockroach#109752][#109752]
```

Comment:
```suggestion
- The `difference` [built-in function]({% link v23.1/functions-and-operators.md %}) had its return type incorrectly set to [`string`]({% link v23.1/string.md %}) instead of [`integer`]({% link v23.1/int.md %}). [#109752][#109752]
```

Reply:
Fixed